src/test/java/com/google/devtools/build/lib/bazel/repository/posix/BUILD

```diff
@@ -0,0 +1,55 @@
+package(
+    default_testonly = 1,
+    default_visibility = ["//src:__subpackages__"],
+)
+
+filegroup(
+    name = "srcs",
+    testonly = 0,
+    srcs = glob(["**"]),
+    visibility = ["//src/test/java/com/google/devtools/build/lib/bazel/repository:__pkg__"],
+)
+
+java_test(
+    name = "RepositoryTests",
```

Comment:
Why is this RepositoryTests when the package is posix? Maybe PosixRepositoryTests?

Reply:
Done.